brainscore/benchmarks/domain_transfer_analysis.py

```diff
@@ -0,0 +1,301 @@
+
+BIBTEX = """
+@inproceedings{bagus2022primate,
+  title={Primate inferotemporal cortex neurons generalize better to novel image distributions than analogous deep neural networks units},
+  author={Bagus, Ayu Marliawaty I Gusti and Marques, Tiago and Sanghavi, Sachi and DiCarlo, James J and Schrimpf, Martin},
+  booktitle={SVRHM 2022 Workshop@ NeurIPS},
+  year={2022}
+}
+"""
+
+
+#### Analysis Benchmark Implementation  ####
+
+class _OOD_AnalysisBenchmark(BenchmarkBase):
+    def __init__(self, metric):
+
+        self._metric = metric
+        self._fitting_stimuli = get_stimulus_set('sanghavi_dicarlo_stimulus_set_domain_transfer')
+        self._fitting_stimuli.identifier = 'domain_transfer_pico_oleo'
+        self._visual_degrees = 8
+        super(OOD_AnalysisBenchmark, self).__init__(
+            identifier='dicarlo.OOD_Analysis_Benchmark',
+            version=1,
+            ceiling_func=lambda: self._metric.ceiling(self._assembly),
+            parent='behavior',
+            bibtex=BIBTEX,
+        )
+
+    # The __call__ method takes as input a candidate BrainModel and outputs a similarity score of how brain-like
+    # the candidate is under this benchmark.
+    # A candidate here could be a model such as CORnet or brain-mapped Alexnet, but importantly the benchmark can be
+    # agnostic to the details of the candidate and instead only engage with the BrainModel interface.
+    def __call__(self, candidate: BrainModel):
+        # based on the visual degrees of the candidate
+        fitting_stimuli = place_on_screen(self._fitting_stimuli, target_visual_degrees=candidate.visual_degrees(),
+                                          source_visual_degrees=self._visual_degrees)
+        candidate.start_recording('IT', time_bins=[(70, 170)])
+        activations = candidate.look_at(fitting_stimuli)
+        activations = activations.squeeze()
+        activations = activations.transpose('presentation', 'neuroid')
+
+        crossdomain_data_dict = get_crossdomain_data_dictionary(domain_transfer_data=activations)
+        
+        crossdomain_results = {}
+        for split in tqdm(np.arange(NUM_SPLITS)):
+            crossdomain_test_images_dict, complete_training_data = split_training_test_images(crossdomain_data_dictionary=crossdomain_data_dict)
+
+            # Train the decoder #
+            decoder = get_decoder(data=complete_training_data, estimator=self._metric)
+            for crossdomain in crossdomain_test_images_dict.keys():
+                test_accuracy = get_classifier_score_2AFC(classifier=decoder, data=crossdomain_test_images_dict[crossdomain])
+                if crossdomain not in crossdomain_results:
+                    crossdomain_results[crossdomain] = [test_accuracy]
+                else:
+                    crossdomain_results[crossdomain].append([test_accuracy])     
+
+        for key, values in crossdomain_results.items():
+
+            crossdomain_results[key] = [np.mean(values)[0], np.std(values)[0]]
```

Comment:
the `[0]` is confusing -- `np.mean` and `np.std` should return a scalar already?

Reply:
they return `array([scalar])`